test-apps/ui-test-app/src/frontend/appui/statusfields/editing/PushPullStatusField.tsx

```diff
@@ -46,29 +45,34 @@ class SyncManager {
     this.changesetListenerInitialized = true;
 
     if (this.briefcaseConnection) {
-      const iModelId = this.briefcaseConnection.iModelId;
+      // const iModelId = this.briefcaseConnection.iModelId;
       try {
-        const accessToken = await IModelApp.getAccessToken();
         // Bootstrap the process by finding out if there are newer changesets on the server already.
-        this.state.parentChangesetId = this.briefcaseConnection.changeset.id;
```

Comment:
Same comment about the commented out code (probably should not be left this way) but I see that this uses events and subscriptions so I wonder if keeping this whole class makes sense.